### PR TITLE
Don't compare completion item details

### PR DESCRIPTION
**Pull request by fivetran-dmitryboykov**
Enable auto retry for BigQuery client which is used to initialize destination;
Log destination initialization error;
Log macro autocompletion e2e test results;

### CODE REVIEW

File: e2e/src/completion/completion_macros.spec.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ suite('Should suggest macros completions', () => {
     // assert
     const expectedCompletions = getMacrosCompletionList();
     expectedCompletions.forEach(c => {
-      const actualCompletion = actualCompletionList.items.find(a => a.label === c.label && a.insertText === c.insertText && a.detail === c.detail);
+      const actualCompletion = actualCompletionList.items.find(a => a.label === c.label && a.insertText === c.insertText);
```
[the remark this answers is not in the document]
Comment (fivetran-dmitryboykov) [reply e2e/src/completion/completion_macros.spec.ts]: Good point. Now just logging added.

File: e2e/src/runTest.ts [hunk 2]
```diff
@@ -9,8 +9,19 @@ import { Client } from 'pg';
 // Expected parameter: path to the folder with the extension package.json
 async function main(): Promise<void> {
   try {
-    await prepareBigQuery();
-    await preparePostgres();
+    try {
```
Comment (pgrivachev) [inline e2e/src/runTest.ts:12]: I think we don't need nested try/catch. You can just add log after prepareBigQuery:
```
await prepareBigQuery();
console.log('bigquery prepared successfully')
await preparePostgres();
```

**Review by pgrivachev — APPROVED** [review]
Looks good, but nested try/catch complicate the code, I'd suggest to avoid it.

**Review by pgrivachev — APPROVED** [review]
👍

**Pull request merged**